runtime/onert/backend/train/TensorRegistry.h

```diff
@@ -82,7 +85,7 @@ class TensorRegistry : public PortableTensorRegistryTemplate<Tensor>
     return PortableTensorRegistryTemplate<Tensor>::setMigrantTensor(ind, tensor);
   }
 
-  void setNativeTensor(const ir::OperandIndex &ind, std::unique_ptr<Tensor> tensor)
+  void setNonConstTensor(const ir::OperandIndex &ind, std::unique_ptr<Tensor> tensor)
```

Comment:
Please update exception message at line 94.

Also, before setting non-const tensor, it checks `_trainable`.
Does every non-const tensor is in _trainable?
I guess there has been non-const tensors before trainable tensor is introduced.

Reply:
> Please update exception message at line 94.

OK, I will change "native" to "non const"

> Also, before setting non-const tensor, it checks _trainable.
> Does every non-const tensor is in _trainable?

No, It checks whether the index is valid. Every const tensor is in `_trainable`.